### PR TITLE
completed css-three-column-layout

### PR DESCRIPTION
**Pull request by m-Cho1**
image

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+.display {
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.column-full {
+  display: flex;
+  justify-content: center;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:31]: The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.column-third {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 50%;
+  align-items: center;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:38]: The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by m-Cho1** [comment]
image

**Comment by m-Cho1** [comment]
edited 3/27/2022

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -27,37 +30,46 @@ body {
 }
 
 .column-full {
-  display: flex;
-  justify-content: center;
+  width: 100%;
 }
 
-.column-third {
+.flex-basis-column {
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:36]: How about `column-half`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by m-Cho1** [comment]
edited! :D

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**